local_planner/src/nodes/local_planner_node_main.cpp

```diff
@@ -50,17 +50,9 @@ int main(int argc, char** argv) {
     if (since_last_cloud > pointcloud_timeout_land &&
         since_start > pointcloud_timeout_land) {
       if (!landing) {
-        mavros_msgs::SetMode mode_msg;
-        mode_msg.request.custom_mode = "AUTO.LOITER";
         landing = true;
```

Comment:
Since we are now not handling landing/loiter etc, can we get rid of this boolean altogether?

Reply:
hmm I don't see a way. We still need it to decide whether to send out stuff or not. The Firmware will only go to loiter if we stop sending. But the name of this variable is not outdated (for the second time) meybe we should rename the bool to aborting?

Reply:
or even more verbose `planner_is_healthy` ? (Will of course flip the true/false semantic of it). I like booleans to have "is" in their name, making it clear that they are booleans